nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    loaded_model = mlflow.pytorch.load_model(model_uri)
</pre><p>What does this load? The neuralforecast object or the NBEATSx model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I believe it loads the model object.

Reply:
Hello @jmoralez,

Is it possible to split this issue in two incremental examples? We first could share this code to show how to use, train, track, monitor, save and load the models using MLFlow? These is very useful already when you are experimenting and tuning hyper parameters and trying multiple models and comparing their performance, MLFlow is great for all of these things as you may know; just leaving loading the model for a later release?

For your consideration!

Cheers.

Reply:
Hey, yeah sorry I thought I had asked for that the last time. We should limit this example to just train, track and monitor. We can then work on saving and loading on our side and when it's ready we can incorporate that to this tutorial or a different one.

Reply:
Hello @jmoralez ,

Great! sounds like a plan.
I just removed the load and predict code sections (for now via commit https://github.com/Nixtla/neuralforecast/pull/892/commits/1e6542ae5c4a12bd999afed263a9a04d5ae7a8ee), so we can publish this Jupyter notebook example as a Neuralforecast and MLFlow integration to train, track and monitor. Please feel free to merge this PR as discussed.

Thank you for your support!